<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="containerPreview">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>81</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::FieldsStayAtSizeHint</enum>
    </property>
    <property name="horizontalSpacing">
     <number>2</number>
    </property>
    <property name="verticalSpacing">
     <number>2</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>R</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="numR">
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>G</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="numG">
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="numB">
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>A</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="numA">
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::FieldsStayAtSizeHint</enum>
    </property>
    <property name="horizontalSpacing">
     <number>2</number>
    </property>
    <property name="verticalSpacing">
     <number>2</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>H</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="numH">
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="maximum">
       <double>360.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>S</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="numS">
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="numV">
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>V</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="containerPalette">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>441</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>91</width>
     <height>66</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="groupConfirmButtons">
    <item>
     <widget class="QPushButton" name="buttonOK">
      <property name="text">
       <string>OK</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonCancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>136</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="groupCopyButtons">
    <item>
     <widget class="QToolButton" name="buttonCopyRGB">
      <property name="text">
       <string>RGB</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="buttonCopyHSV">
      <property name="text">
       <string>HSV</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="buttonCopyHEX">
      <property name="text">
       <string>HEX</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="containerMain" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>251</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QFrame" name="containerColorPlane">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="containerHueSlider">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>20</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>200</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="containerAlphaSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>20</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="buttonScreenPick">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>P</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_3">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::FieldsStayAtSizeHint</enum>
    </property>
    <item row="0" column="1">
     <widget class="QLineEdit" name="textHex">
      <property name="text">
       <string>#FFFFFF</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Hex</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>buttonOK</tabstop>
  <tabstop>buttonCancel</tabstop>
  <tabstop>numR</tabstop>
  <tabstop>numG</tabstop>
  <tabstop>numB</tabstop>
  <tabstop>numA</tabstop>
  <tabstop>numH</tabstop>
  <tabstop>numS</tabstop>
  <tabstop>numV</tabstop>
  <tabstop>textHex</tabstop>
  <tabstop>buttonCopyRGB</tabstop>
  <tabstop>buttonCopyHSV</tabstop>
  <tabstop>buttonCopyHEX</tabstop>
  <tabstop>buttonScreenPick</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
